server/lib/config/nginx.conf

```diff
@@ -88,10 +88,12 @@ http {
 
         location = /404.html {
             internal;
+            client_max_body_size     100G;
```

Comment:
According to google, you can set this in `http` context (all virtual servers), `server` context (a particular site), or `location` (a particular "directory"). Seems we probably want this in `http` or `server` context. Moving the single definition up out of the `location /api` to the enclosing `server` block would probably be fine since we only have one; but it could be hoisted out to the `http` block so it's "even more globalish".

Reply:
Yeah, I don't like the duplication much, either (and I especially don't like the repeated _literals_!), but I concluded that 100G is really big and I don't think we want it permitted anywhere except where we actually need it.  (E.g., I don't think we need or want it for `/dashboard/`, `/static/`, or `/`...and, arguably, we want it only for `/api/vi/upload` and not all of `/api`, but I'm hesitant to split that.)

So, we need it for `/404.html` (I think...) and `/50x.html` (I think!), so I added it to those rather than making it more globalish.

Reply:
OK, I think I found a nice solution; now to see if it works....